gallery/tutorials/tutorials/micrograph_source.py

```diff
@@ -0,0 +1,220 @@
+"""
+=================
+Micrograph Source
+=================
+
+This tutorial will demonstrate how to set up and use ASPIRE's ``MicrographSource`` class.
+"""
+
+import numpy as np
+
+from aspire.image import Image
+from aspire.noise import WhiteNoiseAdder
+from aspire.operators import RadialCTFFilter
+from aspire.source import MicrographSource, Simulation
+
+# %%
+# Creating a Micrograph Source
+# ----------------------------
+# We need to generate our projections by creating a ``Simulation`` and passing it into our MicrographSource
+
+# Let's create our Simulation with a particle box size of 64 and one volume.
+n_particles_per_micrograph = 4
+n_micrographs = 4
+
+sim = Simulation(
+    L=64,
+    n=n_particles_per_micrograph * n_micrographs,
+    seed=1234,
+    C=1,
+    amplitudes=1,
+    offsets=0,
+)
+
+# %%
+# We'll pass our Simulation as an argument and give our MicrographSource other arguments.
+# In this example, our MicrographSource has 4 micrographs of size 500, each with 4 particles.
+micrograph = MicrographSource(
```

Comment:
Since `micrograph` is used below to refer to the actual images, maybe it makes more sense to call this `src`?